File: python/fizzbuzz.py
for i in range(1,100):
    if (i % 15 == 0):
        print(str(i) + ": fizzbuzz")
    elif (i % 3 == 0):
        print(str(i) + ": fizz")
    elif (i % 5 == 0):
        print(str(i) + ": buzz")
    else:
        print(i)

File: c_programming/fizzbuzz.c
#include <stdlib.h>
#include <stdio.h>

int main() {
int i;
for (i=1; i<100; i++) {
    if (i%15 == 0 ) {
        printf("%d: fizzbuzz\n", i);
    }
    else if (i%3 == 0 ) {
        printf("%d: fizz\n", i);
    }
    else if (i%5 == 0 ) {
        printf("%d: buzz\n", i);
    }
    else {
         printf("%d\n", i);
    }
    }
}


